blocks.md

```diff
@@ -1319,10 +1316,17 @@ interval = 1800
 
 Key | Values | Required | Default
 ----|--------|----------|--------
-`bytes` | Whether to use bytes or bits in the display (true for bytes, false for bits). | No | `false`
+`format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{ping}{speed_down}{speed_up}"`
 `interval` | Update interval in seconds. | No | `1800`
-`speed_digits` | Number of digits to use when displaying speeds and latencies. | No | `3`
-`speed_min_unit` | Smallest unit to use when displaying speeds. Possible choices: `"B"`, `"K"`, `"M"`, `"G"`, `"T"`. | No | `"K"`
```

Comment:
Need to document that this has been removed. Perhaps better if we show what the new way is as well.

Reply:
Let's document these breaking change in NEWS.md. I started it off with a couple of examples from other blocks.